edge-modules/load-gen/Program.cs

```diff
@@ -8,15 +8,17 @@ namespace LoadGen
     using Microsoft.Azure.Devices.Client;
     using Microsoft.Azure.Devices.Edge.ModuleUtil;
     using Microsoft.Azure.Devices.Edge.Util;
-    using Microsoft.Azure.Devices.Shared;
     using Microsoft.Extensions.Logging;
     using Newtonsoft.Json;
+    using Message = Microsoft.Azure.Devices.Client.Message;
 
     class Program
```

Comment:
I am going to revert changes to load-gen as they conflict with dylan's incoming changes.